TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Teleop24.java

```diff
@@ -56,21 +88,70 @@ public void loop(){
         telemetry.addData("Right Front Motor Encoder:", frontRight.getCurrentPosition());
 
         telemetry.addData("Right Back Motor Encoder:", rearRight.getCurrentPosition());
+        // Position of elevator arm motor encoder
+        telemetry.addData("Elevator Arm Position:", elevatorPivot.getCurrentPosition());
 
         double drive = (gamepad1.left_stick_y);//inverted???
         double strafe = (-gamepad1.left_stick_x);//inverted???
-        double turn = (gamepad1.right_stick_x);
+        double turn = (gamepad1.right_stick_x);//inverted???
+
+        boolean pivotUp = (gamepad2.y);
+        boolean pivotReset = (gamepad2.a);
+
+        boolean closeGrip = (gamepad2.left_bumper);
+        boolean openGrip = (gamepad2.right_bumper);
+
+        /**
+         </>his is some really janky math that someone implemented back in 2021, but hey, if it works, ¯\_(ツ)_/¯

```

Comment:
Haha; someone is me! >:)

I guess I didn't think about readability when I wrote this. It is pretty difficult to understand if you don't know what signum does. I like the abs adaptation. Maybe even clearer would be `Math.copySign(drive * drive, drive);`?